internal/target/repository.go

```diff
@@ -649,3 +649,15 @@ func (r *Repository) SetTargetHostSets(ctx context.Context, targetId string, tar
 	}
 	return currentHostSets, totalRowsAffected, nil
 }
+
+func fetchLibraries(ctx context.Context, r db.Reader, targetId string) ([]*CredentialLibrary, error) {
```

Comment:
So close to us being perfectly in-sync: 
<img width="877" alt="Screen Shot 2021-05-06 at 2 19 52 PM" src="https://user-images.githubusercontent.com/6704214/117366891-46ac4100-ae76-11eb-8b04-0d281dafa372.png">